pkg/discovery/probe/stitching/stitching_manager.go

```diff
@@ -0,0 +1,201 @@
+package stitching
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"k8s.io/kubernetes/pkg/api"
+
+	"github.com/turbonomic/kubeturbo/test/flag"
+
+	"github.com/turbonomic/turbo-go-sdk/pkg/builder"
+	"github.com/turbonomic/turbo-go-sdk/pkg/proto"
+	"github.com/turbonomic/turbo-go-sdk/pkg/supplychain"
+
+	"github.com/golang/glog"
+)
+
+const (
+	// The property used for node property and replacement entity metadata
+	proxyVMIP   string = "Proxy_VM_IP"
+	proxyVMUUID string = "Proxy_VM_UUID"
+
+	// The default namespace of entity property
+	defaultPropertyNamespace string = "DEFAULT"
+
+	Stitch    stitchingPropertyType = "stitch"
+	Reconcile stitchingPropertyType = "reconcile"
+)
+
+type stitchingPropertyType string
+
+type StitchingManager struct {
+	// key: node name; value: node IP address for stitching.
+	nodeStitchingUIDMap map[string]string
+
+	// key: node name; value: node IP address for stitching.
+	nodeStitchingIPMap map[string]string
+
+	//
+	useVMWare bool
```

Comment:
This is a good point. I've thought about this. The reason why I didn't use "stitchIDType" for flag name is stitching is a term only makes sense inside Turbonomic probe. So it is not a good flag name to expose for users.